### PR TITLE
Feat Added Login & Signup Page

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added Login and Signup Page (Responsive)
Added Input Component.

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
-import { BrowserRouter as Router } from "react-router-dom";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import "./index.css";
 import Navbar from "./Components/Navbar";
 import MiniNav from "./Components/MiniNavbar";
+import Home from "./Pages/Home";
```
Comment (ArjunGTX) [inline frontend/src/App.js:5]: Maintain an `index.js` at the root level of components and pages folder and export each component through this file. The imports will be much cleaner and maintainable

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+import Login from "./Pages/Login";
```
Comment (ArjunGTX) [inline frontend/src/App.js:6]: And also default exports are considered a kind of bad practice. But its completely upto you. Just a suggestion

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
+import Signup from "./Pages/Signup";
 function App() {
   return (
     <Router>
       <MiniNav />
       <Navbar />
+      <Routes>
+        <Route path="/" exact={true} element={<Home />} />
```
Comment (ArjunGTX) [inline frontend/src/App.js]: `exact` is no more used in `react-router-v6`

File: frontend/src/Components/Input/index.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { motion } from "framer-motion";
+import "./style.css";
+
+const Input = ({ onChange, label, ...rest }) => {
+  return (
+    <motion.div>
+      <label>{label}</label>
```
Comment (ArjunGTX) [inline frontend/src/Components/Input/index.jsx:7]: `label` can be linked with corresponding `input` for better accessibility

**Review by ArjunGTX — COMMENTED** [review]
Code looks good. Have suggested some changes. Got some new inputs as well.

File: frontend/src/App.js (hunk 1 above)
Comment (spraveenofficial) [reply frontend/src/App.js:5]: There was only one component in it. That's why I have exported as default.

File: frontend/src/App.js (hunk 3 above)
Comment (spraveenofficial) [reply frontend/src/App.js]: Yes just got to know about that.

File: frontend/src/Components/Input/index.jsx (hunk 4 above)
Comment (spraveenofficial) [reply frontend/src/Components/Input/index.jsx:7]: There is a specific styling for labels while validation. That's why i have used that.

**Pull request merged**